Linked_list/PalindromeCheck.cpp

```diff
@@ -0,0 +1,110 @@
+   /*Check if the linked list is palindrome or not if yes return true if not return false. Linked list has to be taken as an input from the user.*/
+
+#include <iostream>
+class Node
+{
+public:
+	int data;
+	Node *next;
+	Node(int data)
+	{
+		this->data = data;
+		this->next = NULL;
+	}
+};
+
+using namespace std;
+Node* reverse (Node *save)
+       {
+          if(save == NULL || save->next == NULL)  
+           {
+    
+               return save; // BASE CASE: if it is empty or is of size one
+           }
+    
+           Node *new_head = reverse(save->next);//RECURSIVE CALL
+           Node *temp = save->next;      
+           temp->next = save;
+           save->next = NULL;
+           return new_head; 
+       }
+       bool isPalindrome (Node *head)
+       {
+           Node *help=head;
+           Node *temp = reverse(help);//reversing the linked list ans saving its head in temp
+    
+           if(head==NULL){
+               return 1;
+           }
+           while(temp && help){
```

Comment:
add space after while(){ ===> i.e. while() {...

Do this with every if and while conditions in the code